### PR TITLE
Adding class form back

### PR DESCRIPTION
**Pull request by erickpintor**
Closes: https://github.com/fauna/sales-engineering/issues/529

PS: just add the class form. The view that appears once the class is selected will be re-enabled in a different PR.

### CODE REVIEW

File: src/new-dash/schema/components/nav-schema.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import { browserHistory } from "react-router"
 import { Nav } from "office-ui-fabric-react"
 
 import { classesInSelectedDatabase, indexesInSelectedDatabase } from "../"
+import { selectedDatabaseUrl } from "../../router"
 
-const onClick = (e) => e.preventDefault()
+const onClick = (e, link) => {
+  e.preventDefault()
+  browserHistory.push(link.url)
```
Comment (marrony) [inline src/new-dash/schema/components/nav-schema.js]: I'm getting an error in this line, url is undefined

File: src/new-dash/schema/components/class-form.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      this.setState({
+        [field]: !this.state[field]
+      })
+    }
+  }
+
+  onSubmit() {
+    return notify("Class created successfully", dispatch => {
+      let res = dispatch(createClass(
+        this.props.faunaClient,
```
Comment (marrony) [inline src/new-dash/schema/components/class-form.js]: `faunaClient` is coming undefined

**Comment by erickpintor** [comment]
Sorry. I think that was a merge conflict.

File: src/new-dash/schema/components/class-form.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  parseIntOfFail(n) {
+    const res = parseInt(n, 10)
+    if (isNaN(res)) throw Error(`Invalid integer ${n}`)
+    return res
+  }
+
+
+  indexConfig(clazz) {
+    return {
+      name: `all_${clazz.name}`,
```
Comment (marrony) [inline src/new-dash/schema/components/class-form.js]: `clazz.name` is undefined, debugging the correct seems to be `clazz.data.name`

Comment (erickpintor) [inline src/new-dash/schema/components/class-form.js]: fixed by https://github.com/fauna/dashboard/pull/147/commits/cbe85b0a713a264ab14f3927bf0af4f29792a2d9

**Pull request merged**